### PR TITLE
Feat/dev 485 bump gdc ng models

### PR DESCRIPTION
**Pull request by jiakf**

### CODE REVIEW

**Review by ProfOak — APPROVED** [review]
LGTM assuming we merge and tag that branch before merging.

**Review by BenX00 — APPROVED** [review]

**Review by XukunXiangUChicago — APPROVED** [review]
lgtm

**Review by scintillating7 — APPROVED** [review]

**Pull request merged**
